api/src/controllers/authController.ts

```diff
@@ -38,14 +38,47 @@ export const login = catchAsync(async (req: Request, res: Response) => {
   sendSuccess(
     res,
     { token, user: { id: user.id, email: user.email, name: user.name } },
-    "User logged in successfully",
+    "User logged in successfully"
```

Comment:
maybe we might need to align our formatter to get rid of such trailing comma changes